src/FFMpeg/Filters/Video/ResizeFilter.php

```diff
@@ -98,8 +98,10 @@ public function apply(Video $video, VideoInterface $format)
         if (null !== $dimensions) {
             $dimensions = $this->getComputedDimensions($dimensions, $format->getModulus());
 
-            $commands[] = '-s';
-            $commands[] = $dimensions->getWidth() . 'x' . $dimensions->getHeight();
+            // Using Filter to have ordering
+            $commands[] = '-vf';
+            $commands[] = '[in]scale=' . $dimensions->getWidth() . ':' . $dimensions->getHeight() . ' [out]';
```

Comment:
Do we have to put [int] and [out]?
[This doc](https://trac.ffmpeg.org/wiki/Scaling%20(resizing)%20with%20ffmpeg) don't use them.